src/dune_config/dune_config.ml

```diff
@@ -261,7 +261,7 @@ let hash = Poly.hash
 let equal a b = Poly.equal a b
 
 let default =
-  { display = { verbosity = Quiet; status_line = not Config.inside_dune }
+  { display = Scheduler.Config.Display.quiet
```

Comment:
I've changed this so that it is always in quiet mode rather than conditional on being inside Dune.

Reply:
Nevermind, that is nonsense. I have fixed it now.